coverage/clover.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Andrew- 

This looks like a ton of work!  Forgive my ignorance, but Can you. help me understand what these files are doing?

Reply:
Hi @AndrewJuniorB ,  What is the function of this file?

Reply:
I think this shows up after I used 'jest --coverage' not sure. Will need to check it out. Thank you Joe for pointing me there.